packages/manager/apps/pci-kubernetes/src/pages/new/hooks/useEstimationViewModel.ts

```diff
@@ -0,0 +1,65 @@
+import { useMemo } from 'react';
+
+import { NodePoolPrice, TClusterPlan, TClusterPlanEnum, TPlan } from '@/types';
+
+export type EstimationPriceViewModel = {
+  label: string;
+  value?: string;
+  show: boolean;
+};
+
+const useEstimationModel =
```

Comment:
If you abstract hooks in this function, I'll suggest to not export a hook but rather a pure TS function like `selectEstimationPriceFromPlans`